packages/babel-helper-create-class-features-plugin/src/fields.js

```diff
@@ -547,9 +582,7 @@ export function buildFieldsInitNodes(
             privateNamesMap,
           ),
         );
-        staticNodes.push(
-          buildPrivateInstanceMethodDeclaration(prop, privateNamesMap),
-        );
+        staticNodes.push(buildPrivateMethodDeclaration(prop, privateNamesMap));
```

Comment:
Nit: here we should pass `loose` even if it isn't tecnically used for instance methods; but at least if someone changes `buildPrivateMethodDeclaration` to always rely on `loose` they won't unexpectedly get spec-mode output.